.coderabbit.yaml

```
@@ -0,0 +1,14 @@
+reviews:
+  # General instruction for spacing consistency
+  review_instructions: |
+    Always check for consistent indentation and spacing in HTML templates.
+    Ensure all elements at the same nesting level have identical indentation.
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)